test/e2e/upgrades/apps/deployments.go

```diff
@@ -63,15 +63,16 @@ var _ upgrades.Skippable = DeploymentUpgradeTest{}
 // Setup creates a deployment and makes sure it has a new and an old replica set running.
 // This calls in to client code and should not be expected to work against a cluster more than one minor version away from the current version.
 func (t *DeploymentUpgradeTest) Setup(f *framework.Framework) {
-	deploymentName := "deployment-hash-test"
+	deploymentName := "dp"
 	c := f.ClientSet
 	nginxImage := imageutils.GetE2EImage(imageutils.NginxSlim)
 
 	ns := f.Namespace.Name
+	deploymentClient := c.Extensions().Deployments(ns)
```

Comment:
`deploymentClient` is declared as `c.Extensions().Deployments(ns)` appears twice.